src/python_bindings.cpp

```diff
@@ -31,27 +32,18 @@ edge_tuple edgeThreshold(const Eigen::Ref<NumpyMatrix> &distMat,
 }
 
 edge_tuple generateTuples(const std::vector<int> &assignments,
-                            const int within_label,
-                            bool self,
-                            const int num_ref,
-                            const int int_offset) {
-    edge_tuple edges = generate_tuples(assignments,
-                                        within_label,
-                                        self,
-                                        num_ref,
-                                        int_offset);
-    return (edges);
+                          const int within_label, bool self, const int num_ref,
+                          const int int_offset) {
+  edge_tuple edges =
+      generate_tuples(assignments, within_label, self, num_ref, int_offset);
+  return (edges);
 }
 
-edge_tuple generateAllTuples(const int num_ref,
-                                const int num_queries,
-                                bool self = true,
-                                const int int_offset = 0) {
-    edge_tuple edges = generate_all_tuples(num_ref,
-                                            num_queries,
-                                            self,
-                                            int_offset);
-    return (edges);
+edge_tuple generateAllTuples(const int num_ref, const int num_queries,
+                             bool self = true, const int int_offset = 0) {
+  edge_tuple edges =
+      generate_all_tuples(num_ref, num_queries, self, int_offset);
+  return (edges);
 }
```

Comment:
You just hate whitespace, don't you?

Reply:
You need to install the best vs code extension around: https://marketplace.visualstudio.com/items?itemName=shardulm94.trailing-spaces

Reply:
And then once you've accepted that into your life, you can get really pedantic: https://marketplace.visualstudio.com/items?itemName=oderwat.indent-rainbow

Reply:
(I have a lot of extensions)

Reply:
I would inevitably manage to install two extensions with conflicting preferences that would infinite loop as soon as I indented a line